core/src/main/java/org/apache/brooklyn/enricher/stock/Enrichers.java

```diff
@@ -293,42 +307,33 @@ protected String getDefaultUniqueTag() {
             return "aggregator:"+publishing.getName();
         }
         public EnricherSpec<?> build() {
-            Predicate<Object> valueFilter;
```

Comment:
No, it's OK, this is actually fixing broken (although mostly working) code. The code now properly configures the enricher with the passed in `excludingBlank` and `valueFilter` values, and the aggregators that use this functionality manage the configuration of the filter themselves (see `getDefaultVlaueFilter()` method in `Aggregator.java` and the line in `AbstracxtAggregator.java` in `setEntityLoadingConfig()` which uses this to set the `valueFilter` field.) It might be useful to have a check that one of the two config keys, `EXCLUDING_BLANK` or `VALUE_FILTER` is set, though.